src/EverscaleNet.Serialization/JsonOptionsProvider.cs

```diff
@@ -24,7 +24,6 @@ public static class JsonOptionsProvider {
 	private static JsonSerializerOptions CreateJsonSerializerOptions() {
 		var options = new JsonSerializerOptions {
 			MaxDepth = int.MaxValue,
-			Converters = { new JsonStringEnumConverter() },
```

Comment:
it is not default now, generator will add JsonStringEnumConverter only to enums with values